packages/keystone/test.utils.js

```diff
@@ -342,7 +342,10 @@ const makeApolloClient = (serverUrl, opts = {}) => {
         },
         useGETForQueries: true,
         fetch: (uri, options) => {
-            options.headers = { ...options.headers, 'feature-flags': JSON.stringify(Array.from(featureFlagsStore)), ...customHeaders }
+            options.headers = {
+                ...options.headers,
+                'feature-flags': JSON.stringify(Array.from(featureFlagsStore)), ...customHeaders,
+            }
```

Comment:
Do not know about this change... Seems strange to see that one package depends on others :-(

Reply:
I don't understand, what you're writing about?

It's just formatting :)